src/titiler/application/titiler/application/settings.py

```diff
@@ -10,6 +10,7 @@ class ApiSettings(pydantic.BaseSettings):
     cors_origins: str = "*"
     cachecontrol: str = "public, max-age=3600"
     root_path: str = ""
+    path_prefix: str = ""
```

Comment:
@vincentsarago I believe I've covered all configuration changes and tests for API routes:
- `/cog`
- `/stac`
- `/mosaicjson`
- tms
- `/`
- `/healthz`

I *haven't* accounted for the `/docs` endpoint (or docs links). Should I dig into that?